templates/js-cypress/main.js

```diff
@@ -0,0 +1,62 @@
+import { Actor } from 'apify';
+import fs from 'fs';
+import cypress from 'cypress';
+import { globby } from 'globby';
+import log from '@apify/log';
+import 'console.table';
+
+await Actor.init();
+
+const input = await Actor.getInput();
+const runOneSpec = (spec) => cypress.run({ config: input, spec });
```

Comment:
Also, maybe a comment on what exactly it does? This is a template, the more comments and explanations the better